### PR TITLE
SSE-3076 Added few tests related to back link

### PR DESCRIPTION
**Pull request by raghu-kora**
[SSE-3076](https://govukverify.atlassian.net/browse/SSE-3076)

- Added 3 tests related to backlink 
- updated few steps with correct step prefix(Given/When/Then)

[SSE-3076]: https://govukverify.atlassian.net/browse/SSE-3076?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by BorisP-GDS — DISMISSED** [review]

File: acceptance-tests/features/sign-in.feature [hunk 1]
```diff
@@ -12,6 +12,24 @@ Feature: Users can sign in to the self-service experience
       When they submit the email "invalid-email.com"
       Then the error message "Enter an email address in the correct format, like name@example.com" must be displayed for the email field
 
+  Rule: The user navigating back to previous page using back link
```
Comment (MartinH-DI) [inline acceptance-tests/features/sign-in.feature:15]: Suggest: "The user **is** navigating ..."  or "The user navigates ..."

File: acceptance-tests/features/sign-in.feature [hunk 2]
```diff
@@ -12,6 +12,24 @@ Feature: Users can sign in to the self-service experience
...
+    Background:
+      When they submit the email "registered@test.gov.uk"
+    Scenario: User navigating back to /enter-email-address page
```
Comment (MartinH-DI) [inline acceptance-tests/features/sign-in.feature:18]: Suggest: "The user navigates ..."

File: acceptance-tests/features/sign-in.feature [hunk 3]
```diff
@@ -12,6 +12,24 @@ Feature: Users can sign in to the self-service experience
...
+      When they click on the "Back" link
+      Then they should be redirected to the "/sign-in/enter-email-address" page
+
+    Scenario: user navigate back to /enter-password page
```
Comment (MartinH-DI) [inline acceptance-tests/features/sign-in.feature:22]: Suggest: "user navigates ..."

File: acceptance-tests/features/sign-in.feature [hunk 4]
```diff
@@ -12,6 +12,24 @@ Feature: Users can sign in to the self-service experience
...
+      When they click on the "Forgot your password?" link
+      And they click on the "Back" link
+      Then they should be redirected to the "/sign-in/enter-password" page
+
+    Scenario: User navigate back to /forgot-password page
```
Comment (MartinH-DI) [inline acceptance-tests/features/sign-in.feature:27]: Suggest: "user navigates ..."

**Review by MartinH-DI — APPROVED** [review]
Looks good

**Review by BorisP-GDS — APPROVED** [review]

**Pull request merged**